keylime/tpm/tpm_util.py

```diff
@@ -157,8 +157,8 @@ def checkquote(
 
     try:
         verify(pubkey, signature, quote_digest, hashfunc)
-    except InvalidSignature:
-        logger.error("Invalid quote signature!")
+    except InvalidSignature as e:
+        raise Exception("Invalid quote signature!") from e
```

Comment:
At this point I wonder why to catch an specific exception to throw a generic one. Maybe is better to drop the try all together and bubble up the original exception

Reply:
Fixed.